script/paused/deployPausedStaking.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+echo "Instructing the shell to exit immediately if any command returns a non-zero exit status..."
+set -e
+echo "Done."
+
+echo "Navigating to the root directory of the project..."
+cd ../../
+echo "Done."
+
+echo "Setting environment variables..."
+source .env
+echo "Done."
+
+echo "Creating $NETWORK directory inside deployment directory..."
+if [ -z "$NETWORK" ]
+then
+      echo "NETWORK variable inside .env file is not set. Please set NETWORK environment variable."
+      exit 1
+else
+      if [ -d "deployment/$NETWORK" ]
+      then
+            echo "Directory deployment/$NETWORK already exists."
+      else
+            mkdir deployment/$NETWORK
+      fi
+fi
+echo "Done."
+
+echo "Cleaning up the build artifacts to be able to deploy the contract..."
```

Comment:
```suggestion
echo "Cleaning up the build artifacts to be able to deploy the next contract..."
```

To align with others? 🌝

Reply:
Actually first contract deployment is without `next` word and other deployments after it using `next` word.
I think it's like this inside all three scripts for deploying paused versions of smart contracts.